diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1ce7836..0e6fc23 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -127,9 +127,9 @@ if(EXISTS ${CMAKE_SOURCE_DIR}/.git)
 			ERROR_QUIET
 			OUTPUT_STRIP_TRAILING_WHITESPACE
 		)
-		if(NOT ${GIT_DESCRIBE_REVISION} STREQUAL "")
+		if(NOT GIT_DESCRIBE_REVISION STREQUAL "")
 			set(GIT_REVISION "+git-r${GIT_DESCRIBE_REVISION}")
-		endif(NOT ${GIT_DESCRIBE_REVISION} STREQUAL "")
+		endif(NOT GIT_DESCRIBE_REVISION STREQUAL "")
 	endif(GIT_FOUND)
 endif(EXISTS ${CMAKE_SOURCE_DIR}/.git)

diff --git a/data/CMakeLists.txt b/data/CMakeLists.txt
index 369dd69..d1951dc 100644
--- a/data/CMakeLists.txt
+++ b/data/CMakeLists.txt
@@ -38,9 +38,9 @@ if(WITH_GTK)

 	# Install policy rule
 	pkg_get_variable(POLKIT_ACTION_DIR polkit-gobject-1 actiondir)
-	if(${POLKIT_ACTION_DIR} STREQUAL "")
+	if(POLKIT_ACTION_DIR STREQUAL "")
 		set(POLKIT_ACTION_DIR "${CMAKE_INSTALL_FULL_DATADIR}/polkit-1/actions/")
-	endif(${POLKIT_ACTION_DIR} STREQUAL "")
+	endif(POLKIT_ACTION_DIR STREQUAL "")
 	install(FILES
 		${CMAKE_DATA_OUTPUT_DIRECTORY}/io.github.thetumultuousunicornofdarkness.cpu-x-daemon.policy
 		DESTINATION ${POLKIT_ACTION_DIR}
